CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * ``TreeNode.to_array`` now supports replacing ``nan`` branch lengths in the resulting branch length vector with the value provided as ``nan_length_value``.
 * ``skbio.io.format.phylip`` now supports sniffing and reading strict, sequential PHYLIP-formatted files into ``skbio.Alignment`` objects. ([#1006](https://github.com/biocore/scikit-bio/issues/1006))
 * Added `default_gap_char` class property to ``DNA``, ``RNA``, and ``Protein`` for representing gap characters in a new sequence.
+* Replaced `skbio.sequence.IUPACSequence` with `skbio.sequence.GrammaredSequence` and added `skbio.util.classproperty`.
```

Comment:
`skbio.sequence.IUPACSequence` doesn't exist because `IUPACSequence` is private. Can you change the text to:

* Added `skbio.sequence.GrammaredSequence`, which can be inherited from to create grammared sequences with custom alphabets (e.g., for use with `TabularMSA`) ([#1175](https://github.com/biocore/scikit-bio/issues/1175))
* Added `skbio.util.classproperty` decorator